src/webextension/list/manage/components/app.css

```diff
@@ -22,35 +22,25 @@ body > main,
 .app-main {
   display: grid;
   height: 100%;
-  grid-template-columns: 1fr 5fr;
+  grid-template-columns: 2fr 6fr;
   grid-template-rows: auto 1fr;
 }
 
-.side-toolbar,
-.main-toolbar {
+.navigation {
   background-color: #ededf0;
-  border-bottom: 1px solid #c8c8c8;
-}
-
-.side-toolbar {
-  padding-inline-end: 0;
-}
-
-.main-toolbar {
-  padding-inline-start: 0;
+  border-bottom: 1px solid #d1d1d2;
 }
 
 .app-main > aside {
   grid-column: 1;
-  min-width: 0;
+  min-width: 150px;
   min-height: 0;
-  background-color: #dfdfe3;
-  border-right: 1px solid #b1b1b3;
+  background-color: rgba(215, 215, 219, 0.75);
   display: grid;
   grid-template-rows: auto 1fr auto;
 }
 
 .app-main > article {
   grid-column: 2;
-  background-color: #f9f9fa;
+  background-color: #f3f3f5;
```

Comment:
Where's this color coming from? `#f9f9fa` is "Grey 10" in the Photon spec, but I don't see `#f3f3f5` anywhere. (I think I brought this up before in another file.)

Reply:
>I don't see #f3f3f5

Huh.. I'm not sure where it came from. It was somewhere at some point in time..

I currently see it in 3 places and after comparing to Zeplin for this background specifically, it definitely looks like it should be replaced with #f9f9fa.